packages/core-api/lib/versions/2/schema/wallets.js

```diff
@@ -16,7 +16,7 @@ exports.index = {
       secondPublicKey: Joi.string().hex().length(66),
       vote: Joi.string().hex().length(66),
       username: Joi.string(),
-      balance: Joi.number().integer().min(0),
+      balance: Joi.number().integer(),
```

Comment:
Why remove the `min` here, because of the negative genesis balance?

Reply:
Exactly.